litmus-portal/frontend/src/components/FinishModal/index.tsx

```diff
@@ -2,31 +2,27 @@ import Button from '@material-ui/core/Button';
 import Modal from '@material-ui/core/Modal';
 import React from 'react';
 import useStyles from './styles';
-
-/* Icon function is used for finish modal to show mark */
-function Icon() {
-  const classes = useStyles();
-
-  return <img src="icons/finish.png" className={classes.mark} alt="mark" />;
-}
+import ButtonFilled from '../Button/ButtonFilled';
 
 const FinishModal = () => {
   const classes = useStyles();
 
   const [open, setOpen] = React.useState(false);
+  const [modified, setModified] = React.useState(false);
```

Comment:
What is the use of modified here?

Reply:
To close the modal after clicking back to workflow.

Reply:
We already have "open" for closing and opening the modal.

Reply:
Thank you @Jonsy13 I resolved that